docs/src/documentation/03-components/02-structure/table/03-accessibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/table/accessibility/
+---
+
+# Accessibility
+
+## Table
+
+The Table component has been designed with accessibility in mind, and it can be used with keyboard navigation.
+
+### Scope attribute in the TableCell component
+
+If the `as` prop is set to `"th"`, it's possible to set the `scope` prop to identify whether a table header is a column header or a row header.
```

Comment:
Shall we add a link to mdn here?

https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableCellElement/scope